lib/states/game_won_state.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class GameWonState < ConsoleState
```

Comment:
I don't understand why you didn't use namespaces. You have a directory named "states", but you have not created the "States" namespace.

Reply:
Actually, you don't need to use the Game prefix, we are already in the game and we know that.